src/WidgetBase.ts

```diff
@@ -269,6 +284,25 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		return node;
 	}
 
+	@afterRender()
+	protected notifyMetaKeys(node: DNode | DNode[]): DNode | DNode[] {
```

Comment:
Would like to try and avoid using `afterRenders` in `WidgetBase` after analysing the results from `js-framework-benchmark`